src/examples/offboard/offboard_control.cpp

```diff
@@ -36,15 +36,17 @@
  * @addtogroup examples
  * @author Mickey Cowden <info@cowden.tech>
  * @author Nuno Marques <nuno.marques@dronesolutions.io>
+ * @author Mohamed Moustafa <mohamed.moustafa@nxp.com>
  */
 
 #include <px4_msgs/msg/offboard_control_mode.hpp>
-#include <px4_msgs/msg/position_setpoint_triplet.hpp>
+#include <px4_msgs/msg/trajectory_setpoint.hpp>
```

Comment:
Why not vehicle_position_setpoint?

Reply:
You mean vehicle_local_position_setpoint? The type is vehicle_local_position_setpoint but in the code it is ORB_ID(trajectory_setpoint). Actually, I found that setting the vehicle_local_position_setpoint from the agent side will not move the vehicle to the desired location.

Reply:
You are using a rover right?

Reply:
No. I tested this on iris.

Reply:
Hmm. Weird. I have to look at thr offboard changes.

Reply:
@dagar actually pointed that out.

Reply:
So by default we now use trajectory_setpoints to track offboard setpoints?

Reply:
> This is how I understand it. I still wonder why there are some overlaps between position_setpoint and trajectory_setpoint fields like vx, vy, vz, and yaw...

Because we're not finished yet. 😏

Reply:
There might be some message cleanup (https://github.com/PX4/PX4-Autopilot/pull/17094), but I don't have any other major plans for OFFBOARD at the moment.

Getting navigator (position_setpoint_triplet) and the scattered modes under control will gradually continue, but that's now completely detached from OFFBOARD.

### OFFBOARD
 - trajectory_setpoint (position/velocity/acceleration)
 - vehicle_attitude_setpoint (attitude)
 - vehicle_rates_setpoint (angular velocity)